registry/frontend/src/components/ReportAbuse.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import * as Dialog from '@radix-ui/react-dialog';
+import { Cross2Icon } from '@radix-ui/react-icons';
+import './styles.css'
+import {Text, TextArea} from "@radix-ui/themes";
+import * as Checkbox from '@radix-ui/react-checkbox';
+import { CheckIcon } from '@radix-ui/react-icons';
+
+const DialogDemo = () =>{
```

Comment:
Guessing this is still work in progress? As it is called `DialogDemo` and not `ReportAbuse`?

Reply:
Yep